report_a_breach/core/forms.py

```diff
@@ -41,21 +45,34 @@ class Meta:
         fields = ["reporter_email_address"]
 
 
-class EmailVerifyForm(BaseForm):
-    reporter_verify_email = forms.CharField(
+class EmailVerifyForm(BaseModelForm):
+    class Meta:
+        model = ReporterEmailVerification
+        fields = ["email_verification_code"]
+
+    email_verification_code = forms.CharField(
         label=f"{content.VERIFY['text']}",
         help_text=f"{content.VERIFY['helper']}",
     )
 
-    def clean_reporter_verify_email(self):
-        value = self.cleaned_data["reporter_verify_email"]
-        if session_verify_code := self.request.session.get("verify_code"):
-            if value != session_verify_code:
+    def clean(self):
+        cleaned_data = super().clean()
+        value = cleaned_data.get("email_verification_code")
+        verify_timeout_seconds = int(os.getenv("EMAIL_VERIFY_TIMEOUT_SECONDS"))
```

Comment:
related to the above comment, using `env.int()` means you don't have to do the `int()` type casting here

Reply:
please use django.conf.settings (https://docs.djangoproject.com/en/5.0/topics/settings/#using-settings-in-python-code) - it's a nice way that Django allows us to expose settings variable as properties on a settings object.

```suggestion
        from django.conf import settings
        verify_timeout_seconds = settings.EMAIL_VERIFY_TIMEOUT_SECONDS
```